### PR TITLE
implement feedback service

### PR DESCRIPTION
**Pull request by maxxkut**

### CODE REVIEW

File: server/src/main/kotlin/com/github/neho4y/feedback/domain/FeedbackStatus.kt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package com.github.neho4y.feedback.domain
+
+enum class FeedbackStatus {
+
+    CREATED,
+    OPEN,
+    IN_PROGRESS,
+    WAITING,
```
Comment (NeHo4y) [inline server/src/main/kotlin/com/github/neho4y/feedback/domain/FeedbackStatus.kt:8]: What does WAITING status stand for?

File: server/src/main/kotlin/com/github/neho4y/feedback/domain/FeedbackStatus.kt [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+    CLOSED
```
Comment (NeHo4y) [inline server/src/main/kotlin/com/github/neho4y/feedback/domain/FeedbackStatus.kt]: Consider adding RESOLVED status

Comment (maxxkut) [reply server/src/main/kotlin/com/github/neho4y/feedback/domain/FeedbackStatus.kt]: ok

File: server/src/main/kotlin/com/github/neho4y/feedback/domain/FeedbackStatus.kt (hunk 1 above)
Comment (maxxkut) [reply server/src/main/kotlin/com/github/neho4y/feedback/domain/FeedbackStatus.kt:8]: yes

File: server/src/main/kotlin/com/github/neho4y/feedback/service/impl/FeedbackServiceImpl.kt [hunk 3]
```diff
@@ -1,6 +1,75 @@
...
+        val updatedFeedback = feedbackRepository.save(feedback)
+        return updatedFeedback
+    }
+
+    override fun updateStatus(status: FeedbackStatus, id: Long) {
+        val feedback = feedbackRepository.findById(id).orElseThrow()
+        feedback.status = status
+        if (FeedbackStatus.CLOSED == status) {
+            feedback.endDate = LocalDateTime.now()
+        }
```
Comment (NeHo4y) [inline server/src/main/kotlin/com/github/neho4y/feedback/service/impl/FeedbackServiceImpl.kt]: RESOLVED should affect endDate

**Review by NeHo4y — APPROVED** [review]

**Review by dxahtepb — APPROVED** [review]

**Pull request merged**